src/chunk.rs

```diff
@@ -140,7 +140,16 @@ impl Chunk {
         mesher: Box<dyn TilemapChunkMesher>,
         cull: bool,
     ) -> Self {
-        let tiles = vec![None; chunk_size.x as usize * chunk_size.y as usize];
+        let tile_size_x = (1
```

Comment:
I think this should be put inside a static function outside, since I think it is duplicated twice in the project